src/libsyntax/ext/source_util.rs

```diff
@@ -182,9 +182,12 @@ pub fn expand_include_bytes(cx: &mut ExtCtxt, sp: Span, tts: &[tokenstream::Toke
             DummyResult::expr(sp)
         }
         Ok(..) => {
-            // Add this input file to the code map to make it available as
-            // dependency information, but don't enter it's contents
-            cx.source_map().new_source_file(file.into(), String::new());
+            let src = match String::from_utf8(bytes.clone()) {
```

Comment:
You shouldn't need to clone the `bytes` vec.

Reply:
This is cloned because `bytes` is also used to make both the src string and the LitKind::ByteStr ast node at the end of the method.

Reply:
Oh right, I didn't see that. Nevermind then.